src/main/resources/application.yml

```diff
@@ -22,9 +21,11 @@ logging:
         nivio:
           output: TRACE
 
-
 server:
-  port: 8080
+  port: ${PORT:8080}
+
+nivio:
+  baseUrl: ${NIVIO_BASE_URL:}
```

Comment:
Werden die env Variablen irgendwo gesetzt? Im docker-compose?

Reply:
In Heroku. Ich weiß nicht ob die auch ne docker-compose verarbeiten.

Reply:
Passt :)